src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -159,8 +159,15 @@ public int Indent_Braces
 
         public int Indent_CaseContents
         {
-            get { return GetBooleanOption(CSharpFormattingOptions.IndentSwitchCaseSection); }
-            set { SetBooleanOption(CSharpFormattingOptions.IndentSwitchCaseSection, value); }
+            get
+            {
+                return (int)_workspace.Options.GetOption(CSharpFormattingOptions.IndentSwitchCaseSection);
```

Comment:
Do we need to create a new option for back-compat?

I believe if the value was somehow set to `2` we would treat it as `AlwaysIndent` and now we will treat it as `OnlyIndentStatements`.